index.html

```diff
@@ -1,9 +1,21 @@
-<!DOCTYPE html>
-<html>
+<!doctype html>
+<html lang="en">
 
 <head>
-  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <meta charset="utf-8">
+  <meta http-equiv="x-ua-compatible" content="ie=edge">
   <title>Tamarin Prover</title>
+  <meta name="description" content="The Tamarin prover is a security protocol verification tool that supports both falsification and unbounded verification in the symbolic model.">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  
+  <link rel="shortcut icon" href="favicon.ico" type="image/png">
```

Comment:
I think the type should be "image/x-icon", and 'rel="icon"' is better ('shortcut' is legacy supported).